README.md

```diff
@@ -64,6 +66,7 @@ You can use [site environment variables](https://docs.netlify.com/configure-buil
 | `SENTRY_ORG` | The slug of the organization name in Sentry. | - |
 | `SENTRY_PROJECT` | The slug of the project name in Sentry. | - |
 | `SENTRY_RELEASE` | The release ID (a.k.a version). | [COMMIT_REF](https://docs.netlify.com/configure-builds/environment-variables/#git-metadata) |
+| `SENTRY_REPOSITORY` | Override the name of the target Sentry repository. | - |
```

Comment:
@lewisjared Could you change the description to just "The name of the target Sentry repository", the other values in this table don't have actions words like 'override', so we just want to be consistent.

Reply:
Additionally, could you specify the default column to be "Derived from Netlify repository URL" instead of " - ". You mention it is overriding in the description on lines 12-13, so it makes sense to also have a default. Thanks!